test/format/format.h

```diff
@@ -104,7 +104,7 @@ typedef struct {
 	WT_CONNECTION	 *wts_conn;
 	WT_EXTENSION_API *wt_api;
 
-	int   rand_log_stop;			/* Logging turned off */
+	bool  rand_log_stop;			/* Logging turned off */
```

Comment:
Just one clarification required. There are other places in code where it is possible to switch the datatype from int to bool e.g. in function nop_compress, the argument compression_failed. I am assuming that the remaining such variables are for another pull request, not this one.

Reply:
> There are other places in code where it is possible to switch the datatype from int to bool e.g. in function nop_compress, the argument compression_failed.

We converted to`bool` types two years ago (September, 2015). We never found a tool to help, we did it by hand (mostly by searching for integer variables that were set to 0 or 1), and I'm sure we missed some.

Any you find should be updated, if you create the ticket I'm happy to be your reviewer!

That said, `nop_compress` is an external API ([a WT_COMPRESSOR method](http://source.wiredtiger.com/2.9.3/struct_w_t___c_o_m_p_r_e_s_s_o_r.html)), so the `int` wasn't changed to `bool`.